gerritcheck/check.py

```diff
@@ -133,6 +133,12 @@ def cppcheck_on_files(files, commit):
             review["labels"] = {"Code-Review": -1}
             return json.dumps(review)
 
+    # Check the return code only just now as cppcheck might still have returned
+    # some valid comments.
+    if rc != 0:
```

Comment:
What about adding a verbose mode that if enabled shows everything and if disabled shows only negative messages?